code/modules/client/preferences.dm

```diff
@@ -19,6 +19,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	//game-preferences
 	var/lastchangelog = ""				//Saved changlog filesize to detect if there was a change
 	var/ooccolor = null
+	var/enable_tips = TRUE

```

Comment:
I have pref saving and loading under my toggle verbs. 

What do you mean by max version?